src/xcode/ENA/ENA/Source/Scenes/Containers/NavigationControllerWithLargeTitle.swift

```diff
@@ -0,0 +1,34 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+
+class NavigationControllerWithLargeTitle: UINavigationController {
```

Comment:
Would be nice to have a comment here, why we need this custom `UINavigationController`

Reply:
@marcussc I have added the comment.